### PR TITLE
chore: 🍰 Refactor Logos

### LINKED ISSUE #4432: 💥 [DevOps] Ocelot Logo In Header Is Not Replaced By Deployment
## 💥 DevOps ticket

Ocelot logo in header is not replaced by deployment.

### Motive

Should be replaced copying in `webapp.Dockerfile`: https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/blob/master/docker/webapp.Dockerfile

### Related issues

None

### Implementation

Clarify which logo is used there.

### Validation

See deployment.

### Additional context

None

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

- Refactor logos to use same logo file for different places.
- Show all logos with the `logo` component except the maintenance page, which will be done here #4434 .
  - That refactors in the direction of using other type as SVG in the future.

### Issues

- fixes #4432

### Todo

- [x] `webapp/static/img/custom/Logo-Horizontal.svg`: file name still not in small caps?

### CODE REVIEW

File: webapp/components/Logo/Logo.vue [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
   name: 'Logo',
-  components: {
-    svgLogo,
-    svgLogoInverse,
-  },
   props: {
+    /**
+     * Logo type
+     */
+    type: {
```
Comment (Mogge) [inline webapp/components/Logo/Logo.vue:34]: why not call it `logoType`

File: webapp/components/Logo/Logo.vue [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+      type: String,
+      required: true,
+    },
+    /**
+     * Logo type
+     */
+    width: {
```
Comment (Mogge) [inline webapp/components/Logo/Logo.vue:41]: `logoWidth`?

File: webapp/components/Logo/Logo.vue [hunk 3]
```diff
@@ -34,6 +57,39 @@ export default {
...
+        widthDefault: '75%',
+      },
+    }
+    return {
+      logo: logosObject[this.type],
+      metadata,
+    }
+  },
+  computed: {
+    logoWidth() {
```
Comment (Mogge) [inline webapp/components/Logo/Logo.vue:83]: `logoWidthStyle`?

**Review by Mogge — APPROVED** [review]
May be you want to change some variable names. I like that we have the logo component in use for every image now. Great job! Thank you!

File: webapp/components/Logo/Logo.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/components/Logo/Logo.vue]: Nice, done! 🙏🏼

File: webapp/components/Logo/Logo.vue (hunk 2 above)
Comment (Tirokk) [reply webapp/components/Logo/Logo.vue]: Nice, done! 🙏🏼

File: webapp/components/Logo/Logo.vue (hunk 3 above)
Comment (Tirokk) [reply webapp/components/Logo/Logo.vue]: Nice, done! 🙏🏼

**Pull request merged**